src/main/java/com/calendarfiles/Gcal.java

```diff
@@ -110,16 +110,22 @@ public static void printUpcomingEvents(Calendar cal, DateTime now) throws IOExce
         }
     }
 
-    public static void getFreeBusy(Calendar cal, DateTime startTime, DateTime endTime) throws IOException, GeneralSecurityException {
+    public static String getFreeBusy(Calendar cal, DateTime startTime, DateTime endTime) {
       FreeBusyRequest req = new FreeBusyRequest();
       FreeBusyRequestItem item = new FreeBusyRequestItem();
       item.setId("primary");
       List<FreeBusyRequestItem> items = Collections.singletonList(item);
       req.setTimeMin(startTime);
       req.setTimeMax(endTime);
       req.setItems(items);
-      Calendar.Freebusy.Query fbq = cal.freebusy().query(req);
-      FreeBusyResponse fbresponse = fbq.execute();
-      System.out.println("***********free busy:************ \n" + fbresponse.toPrettyString());
+      try {
+        Calendar.Freebusy.Query fbq = cal.freebusy().query(req);
+        FreeBusyResponse fbresponse = fbq.execute();
+        System.out.println("***********free busy:************ \n" + fbresponse.toPrettyString());
```

Comment:
Would recommend deleting this print statement, unless this print statement is used for the core functionality of the app.